app/Http/Controllers/TodoController.php

```diff
@@ -14,9 +14,30 @@ public function index(): JsonResponse
         return response()->json(['todos' => $todos]);
     }
 
-    public function create()
+    public function store(Request $request): JsonResponse
     {
-        // Show the form to create a new Todo
+        $validatedData = $request->validate([
+            'title' => 'required|string',
+            'description' => 'required|string',
```

Comment:
Use array instead of pipe-separated string:
<img width="683" alt="CleanShot 2023-09-14 at 09 30 44@2x" src="https://github.com/Fekete-Balazs710/ToDo_Laravel/assets/74707175/d06c9042-9c64-40dc-abb2-3b1387a620b2">